web/app/js/components/Deployment.jsx

```diff
@@ -188,22 +193,39 @@ export default class Deployment extends React.Component {
     );
   }
 
-  renderContent() {
-    if (_.isEmpty(this.state.metrics)) {
-      return <div>No data</div>;
+  renderDeploymentTitle() {
+    if (!this.state.added){
+      return (<div className="deployment-title">
+        <h1>{this.state.deploy}</h1>
+        <div className="status-badge unadded">UNADDED</div>
+        <div className={"incomplete-mesh-message"}>
+          {this.state.deploy} has not been added to the service mesh
+          <div className="instructions">Add the deployment to the deployment.yml file</div>
+          <div className="instructions">Then run <code>kubectl inject deployment.yml | kubectl apply -f - </code> to add the deployment to the service mesh</div>
+        </div>
```

Comment:
Another small style tweak -- I find the varying line heights in the alert box to be distracting. Ideally all three lines would have the same vertical separation.

![screen shot 2017-12-15 at 12 23 21 pm](https://user-images.githubusercontent.com/9226/34059254-11682c56-e193-11e7-8384-cf6fa90fd65c.png)